loader/src/main/java/net/neoforged/fml/loading/moddiscovery/ModFileParser.java

```diff
@@ -83,4 +84,25 @@ protected static List<String> getMixinConfigs(IModFileInfo modFileInfo) {
             return List.of();
         }
     }
+
+
+    protected static Optional<List<String>> getAccessTransformers(IModFileInfo modFileInfo) {
+        try {
+            final var config = modFileInfo.getConfig();
+            if (config.getConfigElement("accessTransformers").isEmpty()) {
```

Comment:
This causes a crash with the following `mods.toml` entries:
```toml
[[accessTransformers]]
file="ae2_at1.txt"

[[accessTransformers]]
file="ae2_at2.txt"
```

Please fix and test again.